languages/en.ini

```diff
@@ -159,6 +159,7 @@ browse_lcc = "Call Number (LC)"
 browse_publishDate = "Year of Publication"
 browse_title = "Title"
 browse_topic = "Topic"
+Browser = "Browser"
```

Comment:
Related to my comment about Platform -- does "Browser" need its own separate translation, or should we create a single heading translation to capture "Platform / Browser"? (I don't have a strong opinion one way or the other -- just raising the question. It might be more efficient to have just one translation, but more flexible to keep things separate.) If we do keep this, it's another string that might benefit from a more descriptive key.

Reply:
Having them combined to a single translation makes sense. So something like "saved_login_platform_and_browser"?